pogs/pogs_test.go

```diff
@@ -161,13 +161,13 @@ var goodTests = []Z{
 	{Which: air.Z_Which_airport, Airport: air.Airport_lax},
 	{Which: air.Z_Which_grp, Grp: &ZGroup{First: 123, Second: 456}},
 	{Which: air.Z_Which_echo, Echo: air.Echo{}},
-	{Which: air.Z_Which_echo, Echo: air.Echo{Client: capnp.ErrorClient(errors.New("boo"))}},
+	{Which: air.Z_Which_echo, Echo: air.Echo(capnp.ErrorClient(errors.New("boo")))},
 	{Which: air.Z_Which_echoes, Echoes: []air.Echo{
-		{Client: capnp.Client{}},
-		{Client: capnp.ErrorClient(errors.New("boo"))},
-		{Client: capnp.Client{}},
-		{Client: capnp.ErrorClient(errors.New("boo"))},
-		{Client: capnp.Client{}},
+		air.Echo{},
```

Comment:
Nit:  I think this can be simplified to `{}`.  Some linters complain about this, so I vote to reduce such noise.